SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -1,11 +1,16 @@
 namespace SMLHelper.V2.MonoBehaviours
 {
+    using SMLHelper.V2.Assets;
     using UnityEngine;
-    using System.Reflection;
     using Logger = V2.Logger;
 
+    /// <summary>
+    /// This <see cref="MonoBehaviour"/> is <c>automatically</c> added to any <see cref="GameObject"/> created through <see cref="ModPrefab.GetGameObject"/>.
+    /// </summary>
     public class Fixer : MonoBehaviour, IProtoEventListener
```

Comment:
Could there be a better name for this? `Fixer` doesn't say much about what this is supposed to do

Reply:
Clone Remover, Clone Cleaner, Instantiate Deduper, Origin Saver,  etc. 


Modders don't have to ever interact with this class directly under normal usage.  
But it's here in case someone finds a need for it.